spec/features/donuts/02_user_add_new_donut_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+feature "user adds a new donuts", %{
+   As a user
+   I want to be able to add a new donut
+   So that a user can add reviews
+  } do
+  # ACCEPTANCE CRITERIA:
+  # [ X ] Root is Donut#index
+  # [ ] See Add New Donut Link on Root
+  # [ ] On donut#new there should be a Form
+  # [ ] Submitting correct form takes us to donut#show
+  # [ ] Submitting incorrect renders the form with errors
+  before(:each) do
+    visit root_path
+  end
+  scenario "User sees Add New Donut link on root " do
+    expect(page).to have_link "Add New Donut"
+  end
```

Comment:
This test is a bit redundant. It's fine to leave for this project, but in the future I would skip a test this basic. You're inherently testing that it exists in the next couple tests that use that link, so you know it's there. If you were working with a large repo, small tests like this can add up and make running your tests take forever